### DISCUSSION TITLE
Automatically close stale pull requests

### DISCUSSION THREAD

**Discussion by jorenvandeweyer** [node-oauth/node-oauth2-server#110] [General]
I think we should automatically close stale pull requests that are awaiting user input, like #4.

I'd rather have another person creating a new pull request and implement it again, than a stale pull requests, which will never actually be merged, preventing other people from creating another solution.

**Comment by jorenvandeweyer** [comment]
Thoughts? @jankapunkt @jwerre @Uzlopak @HappyZombies

**Reply by jankapunkt** [reply]
I think so, too, however I would check why the PR becomes stale. Some of them were only put on hold, because we did not have the initial workflows and CI set up. So especially #4 should be worth to take a look at it again.
